Readme.rst

```diff
@@ -56,7 +56,7 @@ Basic usage
 Pivot tables are generated by the pivot function, which takes a Model and 3 attribute names,
 to make a pivot table like the example above:
 
->>> pivot_table = pivot(ShirtSales, 'shipped', 'region', 'units')
+>>> pivot_table = pivot(ShirtSales, ['shipped'], 'region', 'units')
```

Comment:
I think we can leave this the way it was since the old API where this is not a list still works.